qualification/Jenkinsfile

```diff
@@ -86,7 +86,7 @@ pipeline {
          sh '''
            spead2_net_raw pytest -v -c qualification/pytest-jenkins.ini qualification\
            --suppress-tests-failed-exit-code \
-           --image-override katgpucbf:harbor.sdp.kat.ac.za/cbf/katgpucbf:latest \
+           --image-override katgpucbf:${katgpucbf_image:-harbor.sdp.kat.ac.za/cbf/katgpucbf:latest} \
```

Comment:
This ... looks very weird. I can't find a reference document anywhere that lets me understand what's going on here. The colon and hyphen - is that Jenkinsfile syntax or something else?

Reply:
It's bash syntax. The bit after the `:-` is used if the variable is not set. Once everything is merged we can probably remove the default, since it's really just there to avoid breaking anything during the transition (since the Jenkins job also has a default).

Reply:
Ah, that makes sense. I hadn't known that about bash.